portality/lib/coerce.py

```diff
@@ -28,7 +32,7 @@ def to_isolang(output_format=None):
         * name
         * fr
     Can be a list in order of preference, too
-    fixme: we could make these pycountry's keys, removing the need for so many transformations and intermediate steps
+    ~~-> Languages:Data~~
```

Comment:
This might be better described as a library - backed by pycountry which uses the Debian maintained dataset for languages.

Reply:
I'm thinking of it as using Language Data in the abstract here, and I've used it like this elsewhere so it ties it all up together.